ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -10,4 +10,27 @@ dot1x protocol lldp bypass
 {%     if dot1x.dynamic_authorization is arista.avd.defined(true) %}
 dot1x dynamic-authorization
 {%     endif %}
+{%     if dot1x.mac_based_authentication or dot1x.radius_av_pair is arista.avd.defined %}
+dot1x
+{%         if dot1x.mac_based_authentication is arista.avd.defined %}
+{%             if dot1x.mac_based_authentication.delay is arista.avd.defined %}
+{%                 set mac_delay_cli = "mac based authentication delay " ~ dot1x.mac_based_authentication.delay ~ " seconds" %}
+   {{ mac_delay_cli }}
+{%             endif %}
+{%             if dot1x.mac_based_authentication.hold_period is arista.avd.defined %}
+{%                 set mac_hold_period_cli = "mac based authentication hold_period " ~ dot1x.mac_based_authentication.hold_period ~ " seconds" %}
+   {{ mac_hold_period_cli }}
```

Comment:
```suggestion
   mac based authentication hold period {{ dot1x.mac_based_authentication.hold_period }} seconds
```

This one is important as there is a typo for `hold_period` instead of `hold period`